src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -215,6 +216,12 @@ public FluentDataGrid()
         columnsFirstCollectedSubscriber.SubscribeOrMove(_internalGridContext.ColumnsFirstCollected);
     }
 
+    /// <inheritdoc />
+    protected override void OnInitialized()
+    {
+        KeyCodeService.RegisterListener(OnKeyDownAsync);
```

Comment:
If you register the service using a method (not `this`) you don't need to add the interface `IKeyCodeListener`.
The interface should be used only when you register the "full" component `this`